media/docs/build/building_with_sycl_support.md

```diff
@@ -67,20 +67,14 @@ $ CC=clang CXX=clang++ cmake .. -G Ninja \
   -DCMAKE_CXX_FLAGS="-ftemplate-backtrace-limit=0 -fdiagnostics-color=always"  
 ```
 
-Note: When building with SYCL support, CMake currently sets the following environment variable to work around an IGC bug:
-```
-$ export IGC_allowDecompose2DBlockFuncs=0
-```
-
 CMake will check that DPC++ compiler is available in the system,
 and it will download the MKL library if it cannot find it.
 
 To build and run a simple PVC gemm example run the commands below.
 
 ```
-$ ninja examples/sycl/pvc/pvc_gemm
-$ cd examples/sycl/pvc/
-$ ./pvc_gemm
+$ ninja 00_pvc_gemm
+$ ./examples/sycl/00_pvc_gemm/00_pvc_gemm
```

Comment:
```suggestion
$ ONEAPI_DEVICE_SELECTOR=level_zero:gpu ./examples/sycl/00_pvc_gemm/00_pvc_gemm
```

Reply:
why do you suggest that. level_zero:gpu is the first pick anyhow so this shouldn't change anything and makes it more verbose for the user.

Reply:
Yes that is correct, I wanted to emphasise on the GPU selection if the first pick for some reason it is not GPU